code/common/subscriptions.q

```diff
@@ -48,14 +48,16 @@ subscribe:{[tabs;instrs;setschema;replaylog;proc]
 	/-if all the tables have been subscribed to on specified instruments
 	if[0=count subtabs; :()];
 	/-if the requested tables aren't available, ignore them and log a message
-	if[count e:subtabs except utabs;
-		.lg.o[`subscribe;"tables ",("," sv string e)," are not available to be subscribed to, they will be ignored"];
+	if[count errtabs:subtabs except utabs;
```

Comment:
utabs is empty here because it is generated from .u.w which isn't defined on the stp. It means `subtabs:subtabs inter utabs` below will set subtabs to ()